### PR TITLE
Feature/pack assets fix

### PR DESCRIPTION
**Pull request by thepian**

### CODE REVIEW

File: test/unit/client/bundler/default.test.js [hunk 1]
```diff
@@ -105,12 +110,19 @@ describe('default bundler:', function () {
...
+      bundler.latestPackedId.should.be.equal('123456789');
+    });
+
   });
 
   describe('client with relative css+code+tmpl', function() {
 
     var client;
 
+    beforeEach(function(done){ fixtures.reset(done); });
```
Comment (paulbjensen) [inline test/unit/client/bundler/default.test.js]: Doesn't need to be changed, but this could be shortened to <code>beforeEach(fixtures.reset);</code>.

**Comment by paulbjensen** [comment]
The tests are passing, the errors are to do with getting Coveralls to receive the coverage stats. Some errors are 500 (a server error on their end), and some look to be an invalid repo token being passed automatically by Travis in the background to coveralls.

Going to merge this in.

**Comment by thepian** [comment]
@paulbjensen I wonder what that 500 error is about, are we behind on coveralls package ?

**Comment by paulbjensen** [comment]
I'll ask their support channel.

**Pull request merged**